pkgs/applications/networking/zeronet/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, pythonPackages }:
+
+pythonPackages.buildPythonApplication rec {
+  version = "0.6.0";
+  name = "zeronet-${version}";
+
+  src = fetchFromGitHub {
+    owner = "HelloZeroNet";
+    repo = "ZeroNet";
+    rev = "v${version}";
+    sha256 = "0qzfm4k63rfxqz72c70d67yrp8jdndsfm891k0drd0prr0fm2fhw";
+  };
+
+  propagatedBuildInputs = with pythonPackages; [ gevent msgpack ];
+
+  phases = [ "unpackPhase" "installPhase" "fixupPhase" ];
```

Comment:
```
format = "other";
buildPhase = ":";
```
The `phases` should typically not be touched as you may loose phases that are relevant.